docs/content/docs/start/tutorial.md

```diff
@@ -60,7 +60,7 @@ Fluid data is stored within containers, and these containers need to be created
 ### Create a new container
 
 The creation section of the application starts with calling `createContainer` and passing in a schema defining which shared objects will be available on the new `container`. After a new container is created, default data can be set on the shared objects before the container is attached to the Tinylicious service. Note, we are passing the parameter "2" in the createContainer call to indicate the version of Fluid Framework that the data
-is compatible with. For new apps always use "2". Use "1" if you have apps running FluidFramework 1 until you've migrated all the apps to version 2.
+is compatible with. For new apps always use "2". Use "1" if you have apps running FluidFramework 1.X until you've migrated all the apps to version 2.X.
```

Comment:
I think @kashms suggestion [from this thread](https://github.com/microsoft/FluidFramework/pull/21586#discussion_r1651795974) still applies here:

> Yea, instead of "until you've migrated all the apps", I would suggest saying "app versions", "app builds" or simply "users". The idea is that you'll have a bunch of old app builds/users on old app builds and you would want to have a defined set of those app versions/users migrated over before flipping the switch.